### PR TITLE
updated new notice , add.tsx in rcid / company and

### LINKED ISSUE #19: [Admin UI] Add Notice, Company, PIO/PPO form
- [Admin Page 9](https://www.figma.com/proto/Tn1nb3bB31R1h4irrGDX1q/SPO-Portal?node-id=73%3A367&scaling=min-zoom&page-id=0%3A1)
  `/admin/rc/:id/notice/new`  
image

- Admin Page 11: `/admin/rc/:id/company/add`  
image

- Admin Page 13: `/admin/rc/:id/pio-ppo/`
image

### PR DESCRIPTION
**Pull request by amdpt323**
index.tsx in pio-ppo

- close #19

### CODE REVIEW

**Comment by ManasGupta1310** [comment]
Rest looks good enough

**Comment by amdpt323** [comment]
@ManasGupta1310 what should the export should be named?

**Comment by ManasGupta1310** [comment]
> @ManasGupta1310 what should the export should be named?

Written on the code  review ...check above

File: pages/admin/rc/[rcid]/company/add.tsx [hunk 1]
```diff
@@ -1,11 +1,46 @@
 import React from 'react'
+import FormControl from '@mui/material/FormControl';
+import { Button, FormHelperText, Input, InputLabel, Stack, TextField, Typography } from '@mui/material';
+import ActiveButton from '../../../../../components/Buttons/ActiveButton';
```
Comment (ManasGupta1310) [inline pages/admin/rc/[rcid]/company/add.tsx:4]: Change this to "@components/Buttons/ActiveButton and same for inactive button

**Comment by ManasGupta1310** [comment]
Looks good! Merging now!

**Pull request merged**